yarn-project/types/src/abi/contract_artifact.ts

```diff
@@ -220,7 +220,7 @@ function getStorageLayout(input: NoirCompiledContract) {
     const name = field.name;
     const slot = field.value.fields[0].value as IntegerValue;
     acc[name] = {
-      slot: slot.value,
+      slot: `0x${slot.value}`,
```

Comment:
This was a bug because then parsing to Fr tried to interpret the string as a normal decimal string due to the lacking "0x" prefix and it failed on value "000000000000000000000000000000000000000000000000000000000000000a".

This didn't occurr before because before this PR we never had a slot larger than 9 and hence the decimal an hex representation was equal.